tests/preload/CMakeLists.txt

```diff
@@ -38,6 +38,11 @@ add_cstyle(tests-preload-config ${CMAKE_CURRENT_SOURCE_DIR}/config/config.c)
 add_check_whitespace(tests-preload-basic ${CMAKE_CURRENT_SOURCE_DIR}/basic/basic.c)
 add_check_whitespace(tests-preload-config ${CMAKE_CURRENT_SOURCE_DIR}/config/config.c)
 
+add_library(setumask SHARED setumask.c)
+set_target_properties(setumask PROPERTIES
+	INCLUDE_DIRECTORIES ${CMAKE_SOURCE_DIR}/src/
+	POSITION_INDEPENDENT_CODE ON)
```

Comment:
Does one really need to manually set POSITION_INDEPENDENT_CODE for a lib marked as SHARED? Isn't that automatic?